src/ucp/proto/proto_init.c

```diff
@@ -216,14 +229,16 @@ ucp_proto_init_memreg_node(ucp_context_h context, ucp_md_map_t reg_md_map,
     if (non_zero_reg_cost_md_map == 0) {
```

Comment:
maybe non_zero_reg_cost_md_map var is not needed anymore, can use *md_map_p

Reply:
Removed.